i18n/pt-br.toml

```diff
@@ -222,4 +222,4 @@ other = "Para verificar a versão, digite "
 other = "Aviso:"
 
 [whatsnext_heading]
-other = "Qual é o próximo"
+other = "O que vem a seguir"
```

Comment:
I recommend saving this cleanup change for a different PR. However, if the difference is because the idiom is said differently in Brazil to in Portugal, that's OK to leave in.

Reply:
Actually the meaning is wrong, but can correct in a followup. I did this small change just to trigger netlify :)